docs/moz-addon-manager.md

```diff
@@ -0,0 +1,204 @@
+# Develop features for mozAddonManager
+
+The AMO and Discovery Pane apps use the
+[`mozAddonManager`](https://bugzilla.mozilla.org/show_bug.cgi?id=1310752)
+web API to achieve a more seamless add-on installation user experience.
+However, this API is only available to a limited list of domains.
+If you go to https://addons.mozilla.org (production) in Firefox then `mozAddonManager`
+is available on the page. If you go to a development site then it's not.
+
+## Turning on mozAddonManager for development
+
+To access `mozAddonManager` on a development site like
+https://addons-dev.allizom.org or https://addons.allizom.org
+go to `about:config` in Firefox and set this property to `true`:
+
+```
+extensions.webapi.testing
+```
+
+Refresh the page, open the JavaScript console, and you should see a global `mozAddonManager` object.
+
+## Developing on Android
+
+The presence of `mozAddonManager` on Firefox for Android will activate the
+add-on installation switch. Again, you will see this on
+https://addons.mozilla.org (production) but not on a development site.
+
+To activate it on a development site, first
+make sure you set the preference up above. If you're testing this on an Android
+device then the preference is enough to access `mozAddonManager`.
+
+If you're testing this on desktop Firefox, you need
```

Comment:
Again I'd remove "this" as it's an unclear pronoun and is easier to read without.